app/models/alchemy/page/page_elements.rb

```diff
@@ -8,7 +8,9 @@ module Page::PageElements
 
       has_many :elements, -> { order(:position) }
       has_many :contents, :through => :elements
-      has_and_belongs_to_many :to_be_sweeped_elements, -> { uniq }, class_name: 'Alchemy::Element', join_table: 'alchemy_elements_alchemy_pages'
+      has_and_belongs_to_many :to_be_sweeped_elements, -> { uniq },
```

Comment:
Rails/HasAndBelongsToMany: Prefer `has_many :through` to `has_and_belongs_to_many`.